### PR TITLE
updateV1.9.0

### PR DESCRIPTION
**Pull request by back0893**
Signed-off-by: liuguojun <back0893@163.com>
update lutos to v1.9.0

### CODE REVIEW

**Comment by jsign** [comment]
@back0893 , we're getting closer! Can you rebase to latest `main` to let the CI run?

**Comment by back0893** [comment]
i run git rebase in my local branch,then commit
is all right?

**Comment by jsign** [comment]
`git rebase master` should be fine, and then force push the branch.

**Comment by back0893** [comment]
force push?
i just commit and push

**Comment by jsign** [comment]
You'll need to `git push --force` for pushing after the rebase. Or just try `git push` and you should see an error mentioning that, give it a try

**Comment by back0893** [comment]
but i just git push,and successful

**Comment by jsign** [comment]
Oh, remember to checkout master, git pull and then rebase. As to update your master branch before rebase to apply the newest changes.

**Comment by back0893** [comment]
i think i do.
i pull origin master ,rebase origin master,and git pull -f

File: wallet/lotuswallet/wallet.go [hunk 1]
```diff
@@ -212,6 +212,11 @@ func (m *Module) Balance(ctx context.Context, addr string) (*big.Int, error) {
 	}
 	b, err := client.WalletBalance(ctx, a)
 	if err != nil {
+		//如果钱包里面灭有钱或者从未交易过.这会导致一个错误,actor not found
+		//查询string中是否有actor not found,但是一般都是矿工
+		if strings.Contains(err.Error(),"actor not found"){
```
Comment (jsign) [inline wallet/lotuswallet/wallet.go:217]: @back0893 , do you mind:
- Removing the comments
- Fixing the formating in this line 217? There should be a space between `,` and `"`, and `)` and `{`.

**Comment by jsign** [comment]
Thanks @back0893 !

**Comment by back0893** [comment]
> 
> 
> Thanks @back0893 !
Thanks @jsign ,this is my first pull merge
Thank you for your patience

**Pull request merged**